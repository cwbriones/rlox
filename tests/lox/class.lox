class Cake {
    init(kind) {
        this.kind = kind;
    }

    bake() {
        print "Baking a " + this.kind + " cake at 300F...";
    }
}

var cake = Cake("chocolate");
cake.bake();
