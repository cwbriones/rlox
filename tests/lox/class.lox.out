Baking a chocolate cake at 300F...
